src/ConnectedMode.UnitTests/Migration/FileCleanerTestCases/AdditionalFiles_my_project_key_Input.xml

```diff
@@ -0,0 +1,29 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<OutputType>Exe</OutputType>
+		<TargetFramework>net6.0</TargetFramework>
+		<ImplicitUsings>enable</ImplicitUsings>
+		<Nullable>enable</Nullable>
+	</PropertyGroup>
+
+	<ItemGroup Label="Should be removed - CSharp">
```

Comment:
All of the items in this group should be removed (although the empty lines will be left behind, along with the whitespace at the start of the line).